lib/tasks/block_level_sync.rake

```diff
@@ -28,6 +28,30 @@ namespace :block_level_sync do
       end
     end
   end
+
+  desc "Bump block level sync for users"
+  task :bump_percentage, [:percentage] => :environment do |_t, args|
+    percentage = args.percentage
+
+    existing_percentage = Flipper[:block_level_sync].percentage_of_actors_value
+    existing_enabled_users = enabled_user_ids.to_set
+
+    ActiveRecord::Base.transaction do
+      Flipper.enable_percentage_of_actors(:block_level_sync, existing_percentage + percentage)
+      Facility.update_all(updated_at: Time.current)
```

Comment:
Do we need to do this? Its going to bust every cache we have across the entire site for any sort of reporting.

Reply:
I changed the cache key for reports, so this should be okay now.

Reply:
@rsanheim It seems like from https://github.com/simpledotorg/simple-server/commit/ba08f5ebabb8670e5bbb6d7ae56225cd63a19e26 we're still using the builtin `cache_key` which will implicitly use an `updated_at`? Am I looking at the wrong thing?

Reply:
No, that's correct, we are using the `updated_at` for the `cache_version` from the facility.  So touching facilities will bust cache keys implicitly...I suppose it isn't a huge deal, as they would all get re-warmed on the next nightly cache warm run.